tools/packaging/static-build/kernel/Dockerfile.cross

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG ARCH=arm64
+FROM dockcross/linux-${ARCH}
+ENV DEBIAN_FRONTEND=noninteractive
+
+# kernel deps
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+	    bc \
+	    bison \
+	    build-essential \
+	    ca-certificates \
+	    curl \
+	    flex \
+	    git \
+	    iptables \
+	    kmod \
+	    libelf-dev \
+	    patch && \
+    if [ "$(uname -m)" = "s390x" ]; then apt-get install -y --no-install-recommends libssl-dev; fi && \
```

Comment:
Could you change this to the following:

```
if echo $ARCH | grep -Eq 390; then apt-get install -y --no-install-recommends libssl-dev; fi && \
```

because an architecture of the image `dockcross/linux-s390x` is `amd64`. Thanks.

Reply:
OK,